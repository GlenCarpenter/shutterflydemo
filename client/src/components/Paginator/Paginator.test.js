import React from "react";
import renderer from "react-test-renderer";
import { Paginator } from "../";

it("renders correctly", () => {
  const tree = renderer.create(<Paginator />).toJSON();
  expect(tree).toMatchSnapshot();
});
