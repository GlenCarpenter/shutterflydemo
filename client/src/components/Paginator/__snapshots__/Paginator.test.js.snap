// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="css-cixvnn-Paginator"
>
  <div>
    Page
  </div>
  <ul
    className="css-qrcrqc-Paginator"
  />
</div>
`;
